### PR TITLE
sign_out

### PR DESCRIPTION
**Pull request by Lundi19**

### CODE REVIEW

File: src/Nav/Navigation.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ import { Link } from "react-router-dom";
 import React from 'react'
 
 export default function Navigation() {
+  
+  function logOut()
+  {
+    sessionStorage.clear();
```
Comment (ArifEbrahim) [inline src/Nav/Navigation.js]: change to local storage

**Review by ArifEbrahim — COMMENTED** [review]
it would be good if you can add a redirect to homepage after signing out using useHistory, thanks!

File: src/Nav/Navigation.js (hunk 1 above)
Comment (Lundi19) [reply src/Nav/Navigation.js]: done

**Review by ArifEbrahim — APPROVED** [review]
:shipit:

**Pull request merged**